credhub.yml

```diff
@@ -0,0 +1,107 @@
+# Update with local file location
+- type: replace
+  path: /releases/-
+  value:
+    name: credhub
+    url: file:///tmp/credhub-0.3.1.tgz
```

Comment:
It can be provided via variable, see example [here](https://github.com/pivotal-cf/pcf-bosh-ci/blob/fa12f47d3946b4dc06c903454cb79909db641695/ops-files/credhub.yml#L6)